### PR TITLE
Sync code between plus and non-plus plugins

### PR DESCRIPTION
**Pull request by HakkyuKim**
Changes:
- mostly code styles
- `device_info` will return empty string when information
  is not available as same as `device_info_plus`

Only `device_info` updates will be published.

### CODE REVIEW

File: packages/connectivity/tizen/src/connectivity_tizen_plugin.cc [hunk 1]
```diff
@@ -40,46 +39,46 @@ class ConnectivityTizenPlugin : public flutter::Plugin {
     }
   }
 
-  void registerObsever(
+  void RegisterObsever(
```
Comment (WonyoungChoi) [inline packages/connectivity/tizen/src/connectivity_tizen_plugin.cc:42]:
```suggestion
  void RegisterObserver(
```

File: packages/connectivity/tizen/src/connectivity_tizen_plugin.cc [hunk 2]
```diff
@@ -40,46 +39,46 @@ class ConnectivityTizenPlugin : public flutter::Plugin {
...
       std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> &&events) {
-    ensureConnectionHandle();
-    if (connection_set_type_changed_cb(connection_, connetionTypeChangedCB,
+    EnsureConnectionHandle();
+    if (connection_set_type_changed_cb(connection_, ConnetionTypeChangedCB,
```
Comment (WonyoungChoi) [inline packages/connectivity/tizen/src/connectivity_tizen_plugin.cc:45]:
```suggestion
    if (connection_set_type_changed_cb(connection_, ConnectionTypeChangedCB,
```

File: packages/connectivity/tizen/src/connectivity_tizen_plugin.cc [hunk 3]
```diff
@@ -40,46 +39,46 @@ class ConnectivityTizenPlugin : public flutter::Plugin {
...
     if (events_ == nullptr) return;
-    std::string replay = convertConnectionTypeToString(state);
+    std::string replay = ConvertConnectionTypeToString(state);
     flutter::EncodableValue msg(replay);
     events_->Success(msg);
   }
 
  private:
-  static void connetionTypeChangedCB(connection_type_e state, void *data) {
+  static void ConnetionTypeChangedCB(connection_type_e state, void *data) {
```
Comment (WonyoungChoi) [inline packages/connectivity/tizen/src/connectivity_tizen_plugin.cc:67]:
```suggestion
  static void ConnectionTypeChangedCB(connection_type_e state, void *data) {
```

File: packages/connectivity/tizen/src/connectivity_tizen_plugin.cc [hunk 4]
```diff
@@ -142,13 +141,13 @@ class ConnectivityTizenPlugin : public flutter::Plugin {
                    std::unique_ptr<flutter::EventSink<>> &&events)
                 -> std::unique_ptr<flutter::StreamHandlerError<>> {
               LOG_INFO("OnListen");
-              this->registerObsever(std::move(events));
+              RegisterObsever(std::move(events));
```
Comment (WonyoungChoi) [inline packages/connectivity/tizen/src/connectivity_tizen_plugin.cc:144]:
```suggestion
              RegisterObserver(std::move(events));
```

**Comment by HakkyuKim** [comment]
@WonyoungChoi 
Thanks for finding! I've also updated `connectivity_plus`.

**Review by swift-kim — APPROVED** [review]

**Pull request merged**